README.md

```diff
@@ -92,29 +98,18 @@ The following third-party libraries are required:
 
 Builds include:
 
-- Operational use GNC build as Wen described for both library and
-  executable (library used for GFS only at this time)
-
-- MRW App uses UPP packaged with nceplibs and cmake to build/run with
-  executable (via release/public-v1 branch).
-
-- SRW App uses UPP repo branch/tag directly and uses cmake to
-  build/run with executable (via release/public-v2 branch).
-
-- Community standalone uses UPP repo branch/tag directly and uses
-  cmake to build/run with executable (via release/public-v2
-  branch). For these procedures, we add a
-  -DCMAKE_PREFIX_PATH=${INSTALL_PREFIX} where INSTALL_PREFIX is the
-  location of the nceplibs installation as a dependency requirement.
-
-```
-mkdir build
-cd build
-cmake .. -DCMAKE_INSTALL_PREFIX=/path/to/install
-(or cmake .. -DCMAKE_INSTALL_PREFIX=/path/to/install -DEMC_EXEC_DIR=ON)
-make -j 4
-make install
-```
+- Inline post (UPP library): Currently only supported for the GFS
+  and MRW Application.
```

Comment:
@fossell In-line post can support RRFS and HAFS,

Reply:
Good catch,  will add those.